manifests/05-deployment.yaml

```diff
@@ -29,4 +29,4 @@ spec:
             - name: OPERATOR_NAME
               value: "elasticsearch-operator"
             - name: PROXY_IMAGE
-              value: "registry.access.redhat.com/openshift3/ose-kube-rbac-proxy:v3.11.69-3"
+              value: "docker.io/jkarasek/rbac-proxy:9"
```

Comment:
there's no other public image?

Reply:
There's outstanding upstream PR. I wanted to see if the latest code, including the proxy change, would pass CI tests

Reply:
moved to quay, yay!